.github/workflows/tests.yaml

```diff
@@ -33,13 +35,34 @@ jobs:
     steps:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
       - uses: actions/setup-python@39cd14951b08e74b54015e9e001cdefcf80e669f # v5.1.1
+        if: ${{ ! matrix.nogil }}
         with:
           python-version: ${{ matrix.python }}
           allow-prereleases: true
           cache: pip
           cache-dependency-path: requirements*/*.txt
+
+      # Free-threading versions of python
+      # https://py-free-threading.github.io/ci/
+      - if: ${{ matrix.nogil && ! matrix.os }}
+        uses: deadsnakes/action@6c8b9b82fe0b4344f4b98f2775fcc395df45e494 # v3.1.0
+        with:
+          python-version: 3.13-dev
+          nogil: true
+      - if: ${{ matrix.nogil && matrix.os == 'windows-latest' }}
+        shell: pwsh
+        run: |
+          $pythonInstallerUrl = "https://www.python.org/ftp/python/3.13.0/python-3.13.0rc1-amd64.exe"
```

Comment:
```suggestion
          $pythonInstallerUrl = "https://www.python.org/ftp/python/3.13.0/python-3.13.0rc2-amd64.exe"
```

Reply:
Done in f4549374fca3c661344595ca35c6ff941b3dd0a2

I had tried this privately just after rc2 was released in the vain hope that it might fix the Windows failure.  (Sadly, it did not.)  I just hadn't pushed it yet.